### PR TITLE
Editor: Bugfix Export of VR-App

### PR DESCRIPTION
**Pull request by Mugen87**
see #8957

### CODE REVIEW

**Comment by Mugen87** [comment]
Currently the following two lines [/editor/js/libs/app.js](https://github.com/Mugen87/three.js/blob/f0828c2757f80ba9b083eed8883347993a0317fb/editor/js/libs/app.js#L133) causes the runtime error, because `VRControls` and `VREffect` are never loaded...

File: editor/js/Menubar.File.js [hunk 1]
```diff
@@ -248,6 +248,18 @@ Menubar.File = function ( editor ) {
 
 		} );
```
Comment (mrdoob) [inline editor/js/Menubar.File.js]: We should only do this if `json.project.vr` is `true` though.

Comment (Mugen87) [inline editor/js/Menubar.File.js]: Um, and what about the `script` tags in `index.html`? If the two files are not present in the published package, the following lines will cause HTTP 404 errors. 

```javascript
<script src="js/VRControls.js"></script>
<script src="js/VREffect.js"></script>
```

Comment (mrdoob) [inline editor/js/Menubar.File.js]: Yeah... I guess we'll have to move that code back to `Menubar.File` and construct it as needed.

Comment (mrdoob) [inline editor/js/Menubar.File.js]: I'll work on that.

Comment (Mugen87) [inline editor/js/Menubar.File.js]: Ok. Should i close this PR?

Comment (mrdoob) [inline editor/js/Menubar.File.js]: Actually... no, I'll continue from here.

**Comment by mrdoob** [comment]
Thanks! 😊

**Comment by Mugen87** [comment]
:blush:👍

**Pull request merged**